sub-rat2/microscopy/sub-rat2_sample-data5_SEM.json

```diff
@@ -0,0 +1,12 @@
+{
+    "PixelSize": 0.093,
+    "FovWidth": 119,
+    "FovHeight": 89,
+    "Species": "rat",
+    "Organ": "spinal cord",
+    "Location": "cervical",
+    "SampleFixation": "4% paraformaldehyde - 0% glutaraldehyde",
+    "Channels": 1,
+    "BitDepth": 8,
+    "Environement": "exvivo"
```

Comment:
```suggestion
    "Environment": "exvivo"
```

Reply:
fix for all json

Reply:
fixed in https://github.com/ivadomed/data_example_microscopy_sem/pull/1/commits/9c14b2792f426b85c24f5723c8003371a0632673